app/models/ab_experiment.rb

```diff
@@ -47,8 +54,9 @@ class AbExperiment < SimpleDelegator
   # @todo If we make heavy use of this class, consider guarding for
   #       valid experiment methods.
   def self.get(experiment:, controller:, user:, default_value:, config: ApplicationConfig)
+    method_name = EXPERIMENT_TO_METHOD_NAME_MAP.fetch(experiment, experiment)
```

Comment:
it might make sense to move this hash lookup into a separate method

```ruby
def method_name_for(experiment)
  EXPERIMENT_TO_METHOD_NAME_MAP.fetch(experiment, experiment)
end
```

Reply:
I like the idea of method extraction for documentation purposes.